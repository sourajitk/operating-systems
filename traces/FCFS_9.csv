1,1,3
2,1,3
3,1,3
4,1,3
5,1,3
6,1,3
7,1,3
8,1,3
9,1,3
10,1,3
11,2,1
12,2,1
13,2,1
14,2,1
15,2,1
16,2,1
17,2,1
18,2,1
19,2,1
20,2,1
