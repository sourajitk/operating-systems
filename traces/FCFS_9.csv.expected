1,4
2,7
3,10
4,13
5,16
6,19
7,22
8,25
9,28
10,31
11,32
12,33
13,34
14,35
15,36
16,37
17,38
18,39
19,40
20,41
